pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -185,6 +190,11 @@ func (c *Cluster) makeDeployment(nodeName string, devices []rookalpha.Device, se
 			"ceph", "osd", "start",
 			"--",
 			"--foreground",
+			"--log-file",
+			"",
+			"--log-to-stderr",
+			"--log-stderr-prefix",
+			"debug ",
```

Comment:
why do we need the `debug` prefix?

Reply:
this was suggested by @liewegas https://github.com/rook/rook/pull/2496#issuecomment-454909537